src/osmf/lib/HDSPlayer.as

```diff
@@ -430,6 +432,7 @@ package
       if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED 
           || _playerState == MediaPlayerState.BUFFERING)
       {
+        _playCompleted = false;
```

Comment:
Hi Sarita,
I think that's a good idea.  
However, also imagine the scenario where the movie URL is changed after a stream completes.  At this point the _playCompleted value will be set to true.  If someone were to query the current time it would return the duration of the movie instead of 0 or the initial time.  This would happen until the user clicks play